### PR TITLE
Only allow wolfram alpha and equation plugin in channels under #help

### PR DESCRIPTION
**Pull request by Armintron**

### CODE REVIEW

File: src/app/plugins/equation.plugin.ts [hunk 1]
```diff
@@ -19,6 +18,11 @@ export class EquationPlugin extends Plugin {
   }
 
   public async execute(message: IMessage, args: string[]): Promise<void> {
+    // Only allow in help channels
+    if (((message.channel as TextChannel).parent as CategoryChannel).name !== 'Help') {
```
Comment (tanndlin) [inline src/app/plugins/equation.plugin.ts:22]: Are you sure the capitalization is correct?
Should probably do `name.toLowerCase() !== 'help'`

File: src/app/plugins/wolframalpha.plugin.ts [hunk 2]
```diff
@@ -24,6 +24,12 @@ export class WolframAlphaPlugin extends Plugin {
   }
 
   public async execute(message: IMessage, args: string[]) {
+    // Only allow in help channels
+    if (((message.channel as TextChannel).parent as CategoryChannel).name !== 'Help') {
```
Comment (tanndlin) [inline src/app/plugins/wolframalpha.plugin.ts:28]: saa

**Review by suneettipirneni — CHANGES_REQUESTED** [review]
It would be preferable if a new `ChannelType` like `ChannelType.Help` was introduced so it could used in other plugins down the line.

**Comment by tanndlin** [comment]
> It would be preferable if a new `ChannelType` like `ChannelType.Help` was introduced so it could used in other plugins down the line.

I thought of that too. Help should be under public as a subset. This PR would need a rename though. Out of scope for this as it stands

**Comment by Armintron** [comment]
Ill leave this PR as is and make another for making channel subsets

**Comment by tanndlin** [comment]
I've started coding this, and changing `Constants.ts` is just not worth the added complexity
We can just make a function to compare a string vs channel parent name if anything

**Comment by tanndlin** [comment]
You can now set a `pluginCategoryName` as of #519
That's all you need to do to solve this PR

**Pull request merged**